app/models/user.rb

```diff
@@ -0,0 +1,37 @@
+require 'bcrypt'
+module ChitterModule
+  module Models
+    class User
+
+      attr_reader :password
+      attr_accessor :password_confirmation
+
+      include DataMapper::Resource
+
+      property :id, Serial
+      property :email, String, required: true
+      property :password_digest, Text
+      property :name, String
+      property :user_name, String, required: true
+
+      has n, :peeps
+
+      validates_confirmation_of :password
+      validates_uniqueness_of :user_name
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.